### ISSUE TITLE
No learning rate decay in the present training code.

### ISSUE THREAD

**Issue by Tommy307** [aim-uofa/DyCo3D#24]
Dear author:
    Thanks for your marvelous job and the code shared in github. I just run your code, and the result is lower than that stated in the paper, and the validation result is shown in the image below:
![image](https://github.com/aim-uofa/DyCo3D/assets/43514798/cb3e27fb-6d8d-4d6a-b575-a7de028cd4a7)  

I find out that in the present train.py, the code that realize lr decay is missing:
![image](https://github.com/aim-uofa/DyCo3D/assets/43514798/24a4c2c4-b635-49f4-85ac-6a063890e853)  
 
I wonder if the code is not right, or you put the lr decay in other place. Looking forward to your reply.

Sincere.
